src/modules/formBuilder/components/itemEditor/conditionals/EnableWhenCondition.tsx

```diff
@@ -217,34 +244,40 @@ const EnableWhenCondition: React.FC<EnableWhenConditionProps> = ({
                 name={`${enableWhenPath}.${index}.answerBoolean`}
                 control={control}
                 shouldUnregister // clear value when un-mounted
+                rules={{ required: true }}
                 render={({
                   field: { ref, ...field },
                   fieldState: { error },
                 }) => (
                   <YesNoRadio
                     label='Value'
                     error={!!error}
-                    helperText={error?.message}
+                    helperText={
+                      error?.type === 'required' ? 'Required' : undefined
```

Comment:
Yeah - I think since this is vertical checkboxes now it looks pretty funky- I would suggest making it a ControlledSelect so that all the comparison "values" have a consistent look. I wanted to test this out before suggestion, so, here is a working approah: https://github.com/greenriver/hmis-frontend/pull/888/files

Reply:
Oh perfect, looks great. Before seeing your comment I put a few minutes into a new `ControlledYesNo` component, but I like your solution much better.

It just needed some tweaking to validation, related to this RHF issue: https://github.com/react-hook-form/react-hook-form/issues/2323

I added that here https://github.com/greenriver/hmis-frontend/pull/885/files#diff-2368e4c27ae82cd3c902453efcfd59df5a1e85dab95f44b903dec4569284ce4eR254-R258